Mathlib/Order/Booleanisation.lean

```diff
@@ -227,7 +227,8 @@ instance instDistribLattice : DistribLattice (Booleanisation α) where
   le_sup_inf x y z := match x, y, z with
     | lift a, lift b, lift c => LE.lift le_sup_inf
     | lift a, lift b, comp c => LE.lift <| by simp [sup_left_comm, sup_comm]
-    | lift a, comp b, lift c => LE.lift <| by simp [sup_left_comm, sup_comm (a := b \ a)]
+    | lift a, comp b, lift c => LE.lift <| by
+      simp [sup_left_comm (a := b \ a), sup_comm (a := b \ a)] -- v4.7.0-rc1 issues
```

Comment:
@mattrobball, I'm not inclined to investigate further. Could you remove the comment?

(Unless your intention was that these comment survive the merge? In which case I think they need to be a link to a tracking issue; I'd prefer to just drop, however.)